scijava/scijava-ops/src/main/java/org/scijava/ops/OpUtils.java

```diff
@@ -382,7 +382,7 @@ private static String paramString(final Iterable<Member<?>> items, final Member<
 
 	/**
 	 * As {@link #paramString(Iterable, Member, String)} with a toggle to
-	 * control if inputs are types only or include the names.
+	 * control if inputs are types only or include the names/descriptions.
```

Comment:
A small detail, but I think `names+descriptions` or similar would read more nicely than a slash.

Reply:
I am not able to find this method (or its javadoc) anymore \*shrugs\*